config/initializers/ab_tests.rb

```diff
@@ -81,4 +81,16 @@ def self.all
       SecureRandom.alphanumeric(8)
     end
   end.freeze
+
+  RECOMMEND_WEBAUTHN_PLATFORM_FOR_SMS_USER = AbTest.new(
+    experiment_name: 'Recommend Face or Touch Unlock for SMS users',
+    should_log: [
+      'Multi-Factor Authentication',
+      'User Registration: MFA Setup Complete',
+      'User Registration: 2FA Setup',
+    ].to_set,
+    buckets: {
+      recommend: IdentityConfig.store.recommend_webauthn_platform_for_sms_ab_test_percent,
+    },
```

Comment:
Request from product to be able to support different configured percentage for recommendation during sign-in vs. account creation.

Could probably handle this by creating separate buckets:

```suggestion
    buckets: {
      recommend_for_account_creation: IdentityConfig.store.recommend_webauthn_platform_for_sms_ab_test_account_creation_percent,
      recommend_for_authentication: IdentityConfig.store.recommend_webauthn_platform_for_sms_ab_test_authentication_percent,
    },
```